### PR TITLE
Guests app integration

### PR DESCRIPTION
**Pull request by nickvergessen**
Bildschirmfoto vom 2022-05-20 14-43-10

- [x] Style to be fixed by @marcoambrosini 

After the Talk bit is merged we can merge:
* https://github.com/nextcloud/guests/pull/892

### CODE REVIEW

File: src/components/RightSidebar/Participants/ParticipantsSearchResults/ParticipantsSearchResults.vue [hunk 1]
```diff
@@ -46,6 +46,18 @@
...
 		</template>
 
+		<template v-if="integrations.length !== 0">
+			<AppNavigationCaption :title="t('spreed', 'Integrations')" />
+
+			<ul>
+				<li v-for="(integration, index) in integrations"
+					:key="'integration' + index"
+					@click="runIntegration(integration)">
+					{{ integration.label }}
```
Comment (nickvergessen) [inline src/components/RightSidebar/Participants/ParticipantsSearchResults/ParticipantsSearchResults.vue:56]: `integration.icon` is available to be used

**Review by PVince81 — APPROVED** [review]
Code looks fine.

I'm just a bit confused that a PR called "Guest integration" doesn't contain the word "guest" anywhere

File: src/components/RightSidebar/Participants/ParticipantsSearchResults/ParticipantsSearchResults.vue [hunk 2]
```diff
@@ -46,6 +46,24 @@
 				@click="handleClickParticipant" />
 		</template>
 
+		<!-- integrations -->
+		<template v-if="integrations.length !== 0">
+			<AppNavigationCaption :title="t('spreed', 'Integrations')" />
+			<ul>
+				<Button v-for="(integration, index) in integrations"
+					:key="'integration' + index"
+					type="tertiary-no-background"
```
Comment (nickvergessen) [inline src/components/RightSidebar/Participants/ParticipantsSearchResults/ParticipantsSearchResults.vue:55]: I would have prefered the avatar style with the circle permanently around it.
But good enough for now

**Comment by nickvergessen** [comment]
Ready to merge @marcoambrosini ?

**Pull request merged**